import os
os.mkdir("fitness")
os.mkdir("checkpoints")
os.mkdir("generations")

os.mkdir("fitness/predator0")
os.mkdir("fitness/predator1")
os.mkdir("fitness/predator2")
os.mkdir("fitness/prey")

os.mkdir("checkpoints/predator0")
os.mkdir("checkpoints/predator1")
os.mkdir("checkpoints/predator2")
os.mkdir("checkpoints/prey")

os.mkdir("generations/predator0")
os.mkdir("generations/predator1")
os.mkdir("generations/predator2")
os.mkdir("generations/prey")
